src/de/gurkenlabs/litiengine/util/io/FileUtilities.java

```diff
@@ -200,4 +200,12 @@ private static boolean isBlackListedDirectory(Path path) {
 
     return false;
   }
+  
+  public static String humanReadableByteCount(long bytes) {
+    int unit = 1024;
+    if (bytes < unit) return bytes + " bytes";
+    int exp = (int) (Math.log(bytes) / Math.log(unit));
+    String pre = new String[] {"kibi", "mebi", "gibi", "tebi", "pebi", "exbi"}[exp-1];

```

Comment:
I would prefer if we went with the correct unit abbreviations for information units (such as `KiB` vs `kibibytes`).
One easy-to-read solution I found was [this one](https://stackoverflow.com/a/66866736/13011276) using unit definitions that build upon each other and offering efficient lookup.